swagger.yaml

```diff
@@ -0,0 +1,249 @@
+swagger: "2.0"
+
+info:
+  title: "Tweet API"
+  description: "This provides an API for creating a message (post), reading a message (get), editing a message (put), delete a message (delete)."
+  version: "1.0"
+
```

Comment:
Compare to our skeleton repo, there are some fields (e.g. `license`, `basePath`, etc.). Could you add them as well?